pipeline/transformer/transformer_mets/src/test/scala/uk/ac/wellcome/platform/transformer/mets/transformer/MetsDataTest.scala

```diff
@@ -458,4 +461,28 @@ class MetsDataTest
     ).toWork(1, Instant.now())
     result shouldBe a[Left[_, _]]
   }
+
+  it("lowercases the b number") {
+    val data = MetsData(
+      recordIdentifier = "B1234"
+    )
+
+    val work = data.toWork(version = 1, modifiedTime = Instant.now()).value
+
+    work.sourceIdentifier shouldBe SourceIdentifier(
+      identifierType = IdentifierType("mets"),
+      ontologyType = "Work",
+      value = "b1234"
+    )
+
+    val mergeCandidates = work.asInstanceOf[Work[Source]].data.mergeCandidates
```

Comment:
Why is `asInstanceOf[..]` required here?

Reply:
Good spot; it isn't. I'll remove that.